api/src/main/java/com/codeforcommunity/ApiMain.java

```diff
@@ -0,0 +1,69 @@
+package com.codeforcommunity;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.http.HttpServer;
+import io.vertx.core.http.HttpServerResponse;
+import io.vertx.ext.web.Route;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.handler.CorsHandler;
+import java.util.Set;
+
+public class ApiMain {
+  public ApiMain() {}
+
+  /** The initialize the sub-router and start the API server. */
+  public void startApi() {
+    Vertx vertx = Vertx.vertx();
+    HttpServer server = vertx.createHttpServer();
+
+    Router router = Router.router(vertx);
+    // Set up cors so that back end and front end can interact through different servers
```

Comment:
elaborate on this... I think there's really no reason this repo isn't 25% comments lol. can discuss at next meeting

Reply:
For CORS specifically, do you think it should be in the repo at this point or should we leave it until we reach whatever module has CORS in it?